### PR TITLE
[MWPW-119603] CKG API Integration Part 2

### PR DESCRIPTION
**Pull request by qiyundai**
Please always provide the [JIRA issue(s)](https://jira.corp.adobe.com/secure/RapidBoard.jspa?rapidView=34618) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix [MWPW-119603](https://jira.corp.adobe.com/browse/MWPW-119603)

**Task description:**
Enhancement to the Link List powered by Browse API
- linklist-qa-mapping.json used for link pill text QA
- Search page now renders linklist pills correctly

**Test URLs:**
Before: https://main--express-website--adobe.hlx.page/express/templates/business-card
After: https://mwpw-119603--express-website--webistry-development.hlx.page/express/templates/business-card?lighthouse=on

### CODE REVIEW

**Review by vhargrave — CHANGES_REQUESTED** [review]
found a few things worth discussing - let me know what you think!

File: express/scripts/api-v3-controller.js
Comment (qiyundai) [reply express/scripts/api-v3-controller.js:41]: Makes perfect sense! The sheet currently is in express/ which is actually a locale directory. In the current setup, they'll need to duplicate this sheet for other locales when they think this feature is ready for other locales(we are not sure at the moment). If we were to move this sheet to the website/ which is the root for all locales, then we could avoid prepending urlPrefix.

**Review by vhargrave — APPROVED** [review]
looking good!

**Comment by vhargrave** [comment]
@qiyundai it looks like there are merge conflicts, but after that we should be good to merge I think

**Comment by qiyundai** [comment]
@vhargrave Might worth taking another look at this point... 😅

File: express/scripts/templates.js
Comment (qiyundai) [reply express/scripts/templates.js:85]: No. PlaceholderRatio is actually not used by the DOM so no need for grammar templating

**Review by JingleH — APPROVED** [review]
looks like people are blocked by this and it is already QAed, so I'm gonna go ahead and approve and merge. @vhargrave and @qiyundai you two feel free to continue any ongoing discussions and make changes later

**Comment by vhargrave** [comment]
> looks like people are blocked by this and it is already QAed, so I'm gonna go ahead and approve and merge. @vhargrave and @qiyundai you two feel free to continue any ongoing discussions and make changes later

@JingleH I think it's good and bad we just merged. So probably for the best you just did it :P

**Comment by vhargrave** [comment]
@JingleH no, but for the future I think we should be a little less trigger happy with the merge button

**Pull request merged**
